docs/wiki/configurator/failsafe-tab.md

```diff
@@ -4,8 +4,125 @@ sidebar_position: 5
+| **Return ground speed (meters/second)** | The speed at which the craft will return to the home position.                                                                                                                                                                                                                                                                                                                                                                                    |
+| **Maximum pitch angle**                 | The maximum pitch angle the craft will use to return to the home position. Rescue throttle usually needs to be increased if the pitch angle is increased. If pitch angle is too low, the craft might not be able to return in high winds.                                                                                                                                                                                                         |
+| **Descent rate (meters/second)**        | The rate at which the craft will descend to the ground when it reaches the home position.                                                                                                                                                                                                                                                                                                                                                         |
+| **Throttle minimum**                    | The **minimum** throttle value the craft will use to return to the home position.                                                                                                                                                                                                                                                                                                                                                                 |
+| **Throttle maximum**                    | The **maximum** throttle value the craft will use to return to the home position.                                                                                                                                                                                                                                                                                                                                                                 |
+| **Throttle hover**                      | The throttle value the craft will use to hover at the return altitude. **It is important to set this value accurately.**                                                                                                                                                                                                                                                                                                                          |
```

Comment:
The initial throttle value that is applied when the rescue first starts. **It is important to set this value accurately.** If the quad drops every time the rescue starts, this value is probably too low.